service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderController.java

```diff
@@ -131,15 +129,14 @@ public void handleSubmitted(@RequestBody CallbackRequest callbackRequest) {
         );
     }
 
-    private Document getDocument(String authorization, String userId, CaseData data, boolean approved)
-        throws IOException {
-        Map<String, Object> cmoDocumentTemplateData = templateDataGenerationService.getTemplateData(data, !approved);
+    private Document getDocument(String auth, String userId, CaseData data, boolean draft) throws IOException {
```

Comment:
do we gain any value renaming from `authorization` to `auth`?

Reply:
it fit on one line 👀